application/views/questionAdministration/modals.twig

```diff
@@ -15,7 +15,7 @@
 
             <div class="modal-body">
                 <p>
-                    {{ gT('Enter one label per line. You can provide a code by separating code and label text with a semicolon or tab. For multilingual surveys you add the translation(s) on the same line separated with a semicolon or tab.') }}
+                    {{ gT('Enter one label per line. You can provide a code by separating code and label text with a semicolon or tab. For multilingual surveys you add the translation(s) on the same line separated with a semicolon or tab. Please recall to use the save button after applying the changes.') }}
```

Comment:
"Please remember"?

Reply:
Isn't the UX severely broken if you have to remind people of using the save-button...?

Reply:
See https://github.com/LimeSurvey/LimeSurvey/pull/2906

Reply:
> "Please remember"?

Done.

> Isn't the UX severely broken if you have to remind people of using the save-button...?

Well, that's another LONG discussion :) This is just for trying to make things less worst :)
Honestly. I don;t think it is a huge UI issue.
I don't believe is something that happens very often.

Reply:
> Honestly. I don;t think it is a huge UI issue.

:+1:

> I don't believe is something that happens very often.

It happen very often, but since it's not an huge issue nobody report it :).